src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
```

Comment:
According to the checklist, you should avoid creating several instances of the same class if you can use only one instance for your purposes. In this case, the variable name 'ball' implies that it is a single ball, but it's being used to represent the Lottery instance that generates random balls. Consider renaming the variable to 'lottery' or similar to better reflect its purpose.